packages/firestore/src/platform_browser/webchannel_connection.ts

```diff
@@ -415,16 +415,6 @@ export class WebChannelConnection implements Connection {
   makeUrl(rpcName: string): string {
     const urlRpcName = RPC_NAME_REST_MAPPING[rpcName];
     assert(urlRpcName !== undefined, 'Unknown REST mapping for: ' + rpcName);
-    const url = [this.baseUrl, '/', RPC_URL_VERSION];
-    url.push('/projects/');
-    url.push(this.databaseId.projectId);
-
-    url.push('/databases/');
-    url.push(this.databaseId.database);
-    url.push('/documents');
-
-    url.push(':');
-    url.push(urlRpcName);
-    return url.join('');
+    return `${ this.baseUrl }/${ RPC_URL_VERSION }/projects/${ this.databaseId.projectId }/databases/${ this.databaseId.database }/documents:${ urlRpcName }`;
```

Comment:
Is there any way to break this up? 159 characters is almost double the allowed line length.

Reply:
yeah sure

Reply:
prettier is reverting my changes

Reply:
FWIW, I think the prior arrangement was essentially a workaround for this disconnect.

What about splitting with a `+`?

Reply:
fixed